tsconfig.json

```diff
@@ -17,7 +17,7 @@
     "moduleResolution": "node",
     "skipLibCheck": true,
     "tsBuildInfoFile": ".tsbuildinfo",
-    "lib": ["es6", "dom", "es2017"]
```

Comment:
Nice. Just out of curiosity, did adding this fix any issues we were having with TS?